src/pr-gating/Setup.md

```diff
@@ -26,17 +26,19 @@
 
 - Setup [build verification for your main branch](https://learn.microsoft.com/en-us/azure/devops/repos/git/branch-policies?view=azure-devops&tabs=browser#build-validation). Pick the CIVerify pipeline. If your developers does not have access rights to dissmiss alerts it is a good idea to set this check to optional. That way, a PR can be completed even if the alert is a false possitive. The rest of the settings can be keep as their default settings.
 
-  The CIVerify pipeline will add PR annotations as comments for the code related to the new alert. Concider setting the requirement 'Check for comment resolution' as a branch policy. This will require all CodeQL generated comments to be concidered before completing the merge. 
+  The CIVerify pipeline will add PR annotations as comments for the code related to the new alert. Concider setting the requirement 'Check for comment resolution' as a branch policy. This will require all CodeQL/Dependency Scanning generated comments to be concidered before completing the merge.
```

Comment:
```suggestion
  The CIVerify pipeline will add PR annotations as comments for the code related to the new alert. Consider setting the requirement 'Check for comment resolution' as a branch policy. This will require all CodeQL/Dependency Scanning generated comments to be concidered before completing the merge.
```